lib/httpi/adapter.rb

```diff
@@ -13,7 +13,7 @@ module Adapter
     ADAPTERS = {}
     ADAPTER_CLASS_MAP = {}
 
-    LOAD_ORDER = [:httpclient, :curb, :em_http, :excon, :http, :net_http, :net_http_persistent]
+    LOAD_ORDER = [:httpclient, :curb, :em_http, :excon, :http, :net_http, :net_http_persistent, :net_http_multipart]
```

Comment:
This `LOAD_ORDER` is actually wrong, because always stop at `:net_http` symbol. In the future this should be deprecated.

Could you change to this `LOAD_ORDER = [:httpclient, :curb, :em_http, :excon, :http, :net_http]`?

Thanks!